frontend/blog/2024-07-10-sunsetting-shields-custom-logos.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: sunsetting-shields-custom-logos
+title: Sunsetting Shields custom logos
+authors:
+  name: PyvesB
+  title: Shields.io Core Team
+  url: https://github.com/PyvesB
+  image_url: https://avatars.githubusercontent.com/u/10694593
+tags: []
+---
+
+Following discussions in [#9476](https://github.com/badges/shields/issues/9476), we've gone ahead and deleted all custom logos that were maintained on the Shields.io side (bitcoin, dependabot, gitlab, npm, paypal, serverfault, stackexchange, superuser, telegram, travis), and from now on will solely rely on the [Simple-Icons project](https://github.com/simple-icons/simple-icons) to provide named logos for our badges.
+
```

Comment:
I think it is worth explaining here that if you were using custom logos, no change is needed. We will just transparently switch you to using the corresponding simple-icon.